src/app/config.service.ts

```diff
@@ -68,6 +75,16 @@ export class ConfigService {
     return this._scanningConfig;
   }
 
+  get appearanceConfig(): AppearanceConfig {
+    // add default appearance config if not present
```

Comment:
Isn't this already done by the way it's declared above?

Reply:
Indeed it has, I have removed it